src/omero_zarr/cli.py

```diff
@@ -287,18 +331,18 @@ def _bf_export(self, image: BlitzObjectWrapper, args: argparse.Namespace) -> Non
             prx, desc = self.client.getManagedRepository(description=True)
             p = image.getImportedImageFilePaths()["server_paths"][0]
             abs_path = Path(desc._path._val) / Path(desc._name._val) / Path(p)
-        target = (Path(args.output) or Path.cwd()) / Path(abs_path).name
+        temp_target = (Path(args.output) or Path.cwd()) / f"{image.id}.out"
```

Comment:
As this led to a few seconds of confusion, the only question is whether this should not be suffixed as `.tmp` instead to clarify this is a temporary directory while the conversion is happening

Reply:
WFM.